.github/workflows/_build-wheel-release-upload.yml

```diff
@@ -159,3 +191,11 @@ jobs:
           TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
         run: |
           twine upload dist/* --verbose
+
+  docs:
+    needs: [github-pre-release, github-release]
```

Comment:
build docs after github (pre) release - should display the tag commit matching that of `main`